Mathlib/CategoryTheory/Comma/Over/Pullback.lean

```diff
@@ -49,7 +49,7 @@ variable [HasPullbacks C]
 
 /-- In a category with pullbacks, a morphism `f : X ⟶ Y` induces a functor `Over Y ⥤ Over X`,
 by pulling back a morphism along `f`. -/
-@[simps! (config := { simpRhs := true}) obj_left obj_hom map_left]
+@[simps! +simpRhs obj_left obj_hom map_left]
```

Comment:
This is also part of #24502; these PRs could land in either order.

Reply:
Thanks: I delegated the other PR.